schema/schema.go

```diff
@@ -6,161 +6,161 @@ type locationSchema struct {
+	URL         string         `jsonschema:"url;description:e.g. http://xyz.example.com - [Related link to the company website];format:uri"`
+	StartDate   string         `jsonschema:"startDate;description:e.g. 2017-06-28 - [resume.json uses the ISO 8601 date standard];format:date"`
+	EndDate     string         `jsonschema:"endDate;description:e.g. 2018-12-29 - [resume.json uses the ISO 8601 date standard];format:date"`
+	Summary     string         `jsonschema:"summary;description:Give an overview of your responsibilities at the company"`
+	Highlights  []string       `jsonschema:"highlights;description:Specify multiple accomplishments;items_description:e.g. Worked with mobile team at Twitter to develop remote debugging tools for mobile browsers"`
 }
 
 type educationSchema struct {
-	Institution string         `jsonschema:"institution,description:e.g. XYZ Institute of Technology - [Add institute name]"`
-	Location    locationSchema `jsonschema:"location,description:e.g. Germany - [Location for this institution],format:location"`
-	Area        string         `jsonschema:"area,description:e.g. Engineering"`
-	StudyType   string         `jsonschema:"studyType,description:e.g. Bachelor"`
-	StartDate   string         `jsonschema:"startDate,description:e.g. 2017-06-29 - [resume.json uses the ISO 8601 date standard],format:date"`
-	EndDate     string         `jsonschema:"endDate,description:e.g. 2013-06-29 - [resume.json uses the ISO 8601 date standard],format:date"`
+	Institution string         `jsonschema:"institution;description:e.g. XYZ Institute of Technology - [Add institute name]"`
+	Location    locationSchema `jsonschema:"location;description:e.g. Germany - [Location for this institution];format:location"`
+	Area        string         `jsonschema:"area;description:e.g. Engineering"`
+	StudyType   string         `jsonschema:"studyType;description:e.g. Bachelor"`
+	StartDate   string         `jsonschema:"startDate;description:e.g. 2017-06-29 - [resume.json uses the ISO 8601 date standard];format:date"`
+	EndDate     string         `jsonschema:"endDate;description:e.g. 2013-06-29 - [resume.json uses the ISO 8601 date standard];format:date"`
 	Score       struct {
-		Scoretype  string `jsonschema:"scoretype,description:eg. GPA or PERCENTAGE - [Add score type]"`
-		Scorevalue string `jsonschema:"scorevalue,description:eg. 3.4/4.0 - [Add obtained score/total score]"`
-	} `jsonschema:"score,additionalProperties"`
-	Courses []string `jsonschema:"courses,description:List notable courses/subjects,items_description:e.g. CS302 - Introduction to Algorithms - [Add course name]"`
-	Honors  []string `jsonschema:"honors,description:List education honours,items_description:e.g. Magna Cum Laude"`
+		Scoretype  string `jsonschema:"scoretype;description:eg. GPA or PERCENTAGE - [Add score type]"`
+		Scorevalue string `jsonschema:"scorevalue;description:eg. 3.4/4.0 - [Add obtained score/total score]"`
```

Comment:
Why not use `ScoreType`, `ScoreValue`, `scoreType` and `scoreValue`?

Reply:
Might actually be shortened to `Type` and `Value` as it's within the `Score` struct. Thoughts?

Reply:
Fixed in bd74350